src/main.rs

```diff
@@ -237,16 +234,17 @@ mod tests {
         let _ = panic::catch_unwind(|| {
             panic!("Oh, noes!");
         });
-
         // Look for the expected backtrace inside the log
-        validate_backtrace(
-            log_file.as_str(),
-            &[
-                // Lines containing these words should have appeared in the log, in this order
-                ("ERROR", "main.rs", "Panic occurred"),
-                ("ERROR", "main.rs", "stack backtrace:"),
-                ("0:", "0x", "backtrace::"),
-            ],
+        assert!(
+            validate_backtrace(
+                log_file.as_str(),
+                &[
+                    // Lines containing these words should have appeared in the log, in this order
+                    ("ERROR", "main.rs", "Panic occurred"),
+                    ("ERROR", "main.rs", "stack backtrace:"),
+                    ("0:", "0x", "firecracker::main::"),
+                ],
+            ) || println!("Could not validate backtrace!\n {:?}", Backtrace::new()) != ()
```

Comment:
nit: you can use `eprintln!`.

Reply:
I though about that but the following thing happens:
Let us say we put a custom message (i.e "Lala") to an assert and make it fail. The following will be printed by "cargo test":
```
---- tests::test_main stdout ----
thread 'tests::test_main' panicked at 'Lala', src/main.rs:202:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
You can see that `cargo test` reports that as *stdout*. This is why I put there `println!` instead of `eprintln!` (stderr).